models/commons/src/discounted-price/discounted-price-draft/builders.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { DiscountedPriceDraftRest, DiscountedPriceDraftGraphql } from './index';
+
+describe('DiscountedPriceDraft Builder', () => {
+  it('should build properties for the discounted price draft REST representation', () => {
+    const restDraftModel = DiscountedPriceDraftRest.random().build();
+
+    expect(restDraftModel).toEqual(
+      expect.objectContaining({
+        value: expect.objectContaining({
+          centAmount: expect.any(Number),
+          currencyCode: expect.any(String),
+        }),
+        discount: expect.objectContaining({
+          id: expect.any(String),
+          typeId: 'product-discount',
+        }),
+      })
+    );
+  });
+
+  it('should build properties for the discounted price draft GraphQL representation', () => {
+    const graphqlDraftModel = DiscountedPriceDraftGraphql.random().build();
+
+    expect(graphqlDraftModel).toEqual(
+      expect.objectContaining({
+        __typename: 'DiscountedProductPriceValue',
+        value: expect.objectContaining({
+          centAmount: expect.any(Number),
+          currencyCode: expect.any(String),
```

Comment:
We're missing the `__typename` here:
```suggestion
          currencyCode: expect.any(String),
          __typename: 'ProductPrice',
```

Reply:
The type name is Money
![image](https://github.com/user-attachments/assets/eba024cb-b849-46cd-b4c7-836e4b492c67)